src/Entity/MessageType.php

```diff
@@ -323,10 +327,13 @@ public function getUuid() {
   public function getText($langcode = NULL, $options = array()) {
     $text = $this->text;
 
-    if ($langcode && \Drupal::moduleHandler()->moduleExists('config_translation')) {
-      $config_translation = \Drupal::languageManager()->getLanguageConfigOverride($langcode, 'message.type.' . $this->id());
-      if ($translated_text = $config_translation->get('text')) {
-        $text = $translated_text;
+    if ($langcode) {
+      $language_manager = \Drupal::languageManager();
+      if ($language_manager instanceof ConfigurableLanguageManagerInterface) {
```

Comment:
Because at least testing is easier  and conceptually, you do not care really how the getLanguageConfigOverride gets there, it violates encapsulation to check for a module. You want *something* to provide getLanguageConfigOverride and don't care about anything else.